### DISCUSSION TITLE
Welcome to MAAD-Attack-Framework Discussions!

### DISCUSSION THREAD

**Discussion by openrec0n** [vectra-ai-research/MAAD-AF#1] [Announcements]
## 👋 Welcome!
Discussions is a place to connect with the members of the community. We hope that you:
  * Ask questions about MAAD-AF.
  * Share ideas.
  * Bring issues to attention.
  * Share interesting use cases that may help others.
  * Engage with other community members.

**Comment by irchv** [comment]
Hey, it can be not a very smart question but when "executing " the ADD CREDS command, I'm prompted to select a token or a password, I choose a token, pasting the needed one but when executing SHOW CREDS nothing shows up. Any Idea what can cause this? Thanks indeed
